tests/unit/services/auth/test_auth_handlers.py

```
@@ -0,0 +1,327 @@
+    from app.services.auth_handlers import OAuthCallbackHandler
+
+    if callback_event is None:
+        callback_event = Event()
+    if callback_lock is None:
+        callback_lock = Lock()
+    if callback_data is None:
+        callback_data = {}
+
+    # We need to patch the handler's initialization since BaseHTTPRequestHandler
+    # tries to handle the request immediately in __init__
+    with patch.object(OAuthCallbackHandler, '__init__', lambda self, *args, **kwargs: None):
```

Ruff (0.14.10)

Unused lambda argument: `self`

(ARG005)